strapi/src/api/page/content-types/page/schema.json

```diff
@@ -132,6 +140,26 @@
         }
       }
     },
+    "headerPrimaryButton": {
+      "type": "component",
+      "repeatable": false,
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "component": "blocks.page-header-button"
+    },
+    "headerSecondaryButton": {
+      "type": "component",
+      "repeatable": false,
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "component": "blocks.page-header-button"
+    },
     "pageButtonContent": {
```

Comment:
This old `pageButtonContent` acted as primary header button. It may need some migration or at least a table of page where it was used.

Reply:
Now it's merged, and `pageButtonContent` is used as a first button when available. I will remove them when migrated.